src/framework/mlt_property.c

```diff
@@ -821,7 +822,7 @@ char *mlt_property_get_string_l_tf( mlt_property self, locale_t locale, mlt_time
 		setlocale( LC_NUMERIC, localename );
 #endif // _WIN32
 
-		if ( self->types & mlt_prop_int )
+		if ( self->types & mlt_prop_int || self->types & mlt_prop_color ) // warning
```

Comment:
Why do you have a warning here?

In the case of color, should it be converted to a hex string?

Reply:
Well the warning comment doesn't belong here. I just added it for me during development and forgot to remove it. Solved.

Not sure about the hex string.

Reply:
I cannot see the context for this anymore, but a hex string is not the preferred or typical format. CSS color string is.